app/api/video/migrations/0001_initial.py

```diff
@@ -15,10 +15,11 @@ class Migration(migrations.Migration):
             name='Video',
             fields=[
                 ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('title', models.CharField(max_length=100)),
-                ('description', models.TextField(max_length=1024)),
-                ('bucket_path', models.CharField(max_length=100)),
-                ('upload_date', models.DateTimeField(auto_now_add=True)),
+                ('title', models.CharField(max_length=100, verbose_name='title')),
```

Comment:
you should never ever change existing migrations in your PR, just add a new one. research why changing migrations is bad